A tutorial on how to create a 3Box chatbox plugin.md

```diff
@@ -0,0 +1,11 @@
+# Content
+A tutorial on how to create a 3Box chatbox plugin
+
+Outline 
+
+1). An overview of 3box chatbox plugin
+2). Introduction to 3Box chatbox plugin
+3). Use cases of 3Box chatbox plugin
+4). How 3box chatbox works
+(i) example of 3box chatbox plugin
+5). How to build a 3box chatbox plugin.
```

Comment:
@writeprovidence What will you build?

Reply:
A feedback system for dapps based on ethereum network.

Reply:
@ChrisChinchilla still waiting for your feedback

Reply:
@ChrisChinchilla , i noticed a change of title to  (A tutorial on how to create an application using 3Box chatbox plugin)

Reply:
Yup, that was at the request of the 3box team, they wanted to make it clearer. Could you add those details to the outline please?